rct229/rules/section21/section21rule4.py

```diff
@@ -0,0 +1,50 @@
+from rct229.data.schema_enums import schema_enums
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+
+APPLICABLE_SYS_TYPES = ["SYS-1", "SYS-5", "SYS-7", "SYS-11.2", "SYS-12", "SYS-1A", "SYS-7A", "SYS-11.2A", "SYS-12A"]
+BOILER_COMBUSTION_OPTION = schema_enums["BoilerCombustionOptions"]
+
+
+class Section21Rule4(RuleDefinitionListIndexedBase):
+    """Rule 3 of ASHRAE 90.1-2019 Appendix G Section 23 (Hot water loop)"""
```

Comment:
This is a really minor one and doesn't affect the code implementation but, I'd appreciate it if you could change it to Rule ```4```.